data/tutorials/en/006_if_statements_loops_and_recursion.md

```diff
@@ -1136,7 +1136,7 @@ The basics of arrays are simple:
 ```ocaml
 # let a = Array.create 10 0
 Line 1, characters 9-21:
-Alert deprecated: Stdlib.Array.create
```

Comment:
Haha this is strange, did we push test output from a non `4.10.2` compiler and there was this small change?

Reply:
Oops, I did a `s/deprecated/deprecate/` in the repo, I shouldn't have commited this file. Will revert this

Reply:
(good catch!)